script.js

```diff
@@ -0,0 +1,229 @@
+
+const TEXTAREA = document.createElement('textarea');
+TEXTAREA.id = 'textarea';
+TEXTAREA.autofocus = true;
+const KEYBOARD = document.createElement('div');
+KEYBOARD.id = 'keyboard';
+const HINT = document.createElement('div');
+HINT.innerText = 'Shift + Alt - change language\n Created under Windows';
+document.body.append(TEXTAREA);
+document.body.append(KEYBOARD);
+document.body.append(HINT);
+let language = sessionStorage.getItem('language') != null ? sessionStorage.getItem('language') : 'en';
```

Comment:
I prefer to store such strings into constant object 
```
const LANGUAGES = {
    en: 'en',
    ru: 'ru',
}
```
and then use it
`let language = sessionStorage.getItem('language') != null ? sessionStorage.getItem('language') : LANGUAGES.en;`

Reply:
fix in commit 95b066e